### PR TITLE
save fix

### PR DESCRIPTION
**Pull request by HerbFargus**

### CODE REVIEW

File: libretro.cpp [hunk 1]
```diff
@@ -112,7 +112,8 @@ void retro_init(void)
    if (environ_cb(RETRO_ENVIRONMENT_GET_SAVE_DIRECTORY, &dir) && dir)
    {
 	  // If save directory is defined use it, otherwise use system directory
-      retro_save_directory = *dir ? dir : retro_base_directory;
+      // retro_save_directory = *dir ? dir : retro_base_directory;
+	  retro_save_directory = dir;
```
Comment (RobLoach) [inline libretro.cpp]: The comments above should be updated, since they no longer apply with this change.

**Comment by twinaphex** [comment]
@hizzlekizzle Can you determine if this PR should still be merged or not?

**Comment by retro-wertz** [comment]
this is not anymore necessary in current code.

**Comment by HerbFargus** [comment]
I'll test your fixes when I get a moment but trust that you've solved it in the mean time by closing this, thanks for your work on this and the other cores

**Comment by retro-wertz** [comment]
ok, i ddnt even notice the posts on the cores were from the same person. feel free to reply back for concerns.

**Pull request closed without merging**